src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+public class Lottery {

```

Comment:
You should create two class variables ColorSupplier and Random and use them to create the balls

Reply:
not fixed

Reply:
not fixed